bench/workgen/workgen.cxx

```diff
@@ -417,8 +675,8 @@ Context::operator=(const Context &other)
 
 ContextInternal::ContextInternal()
     : _tint(), _table_names(), _table_runtime(nullptr), _runtime_alloced(0), _tint_last(0),
-      _dyn_tint(), _dyn_table_names(), _dyn_table_runtime(), _dyn_tint_last(0), _dyn_table_in_use(),
-      _dyn_tables_delete(), _context_count(0), _dyn_mutex(new std::mutex())
+      _dyn_tint(), _dyn_table_names(), _dyn_table_runtime(), _dyn_tint_last(0), _context_count(0),
```

Comment:
Why do we maintain two sets of tables rather than one? I feel like this could be problematic. Are there functions that can and cannot be performed on one set vs the other set? If the user creates one or more tables in their test (i.e. using `Table()`), or if we run a workgen script against an existing database and read in the existing tables, can any of those tables ever be dropped? Do random operations execute on both sets of tables?

Reply:
Discussed how these scenarios can be managed with Sulabh and am happy with the design decision. Ok to resolve.